pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,10 +1,22 @@
+# optimus_v4.0.0
+
+2020-08-10 (Date of Last Commit)
+### Breaking changes to the pipeline inputs. 
```

Comment:
Do we want to demarcate breaking changes moving forward for all pipelines? If so, I can add to the changelog style guide

Reply:
Yes, I think it is a good idea!